pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter_test.go

```diff
@@ -124,6 +124,26 @@ var _ = Describe("Validating VM Admitter", func() {
 		Expect(resp.Result.Details.Causes[0].Field).To(Equal("spec.template.spec.domain.devices.disks[0].name"))
 	})
 
+	It("should allow VM with missing volume disk or filesystem", func() {
```

Comment:
To clarify - this is now allowed because the VMI is being mutated to have the missing disk, right?

Reply:
Correct yes, thus we can now allow VMs to be admitted without a complete mapping of disks/filesystems for their associated volumes.